pages/index.js

```diff
@@ -11,8 +11,8 @@ export default function Home({ entries }) {
     <>
       <Head>
         <title>Juani Gallo - Blog</title>
-        <meta name="description" content="Juani Gallo's blog" />
-        <link rel="icon" href="/favicon.ico" />
+        <meta name='description' content="Juani Gallo's blog" />
+        <link rel='icon' href='/favicon.ico' />
```

Comment:
Please change your linter settings to avoid unexpected changes

Reply:
Changed